src/components/WizardModal/_wizard-modal.scss

```diff
@@ -5,10 +5,6 @@
     min-width: 410px;
   }
 
-  .#{$prefix}--progress {
```

Comment:
what is the reason for removing these styles here?

Reply:
@davidicus see my previous reply https://github.com/IBM/carbon-addons-iot-react/pull/1359#discussion_r453860655